contracts/TroveManager.sol

```diff
@@ -214,6 +214,18 @@ contract TroveManager is VestaBase, CheckContract, ITroveManager {
 			singleLiquidation.collGasCompensation
 		);
 
+		uint256 ICR = getCurrentICR(_asset, _borrower, vestaParams.priceFeed().fetchPrice(_asset));
+		uint256 collPercToSP = 1e18 + vestaParams.BonusToSP(_asset);
+
+		if (ICR > collPercToSP) {
+			uint256 collToBorrower = collToLiquidate.mul(ICR - collPercToSP).div(ICR);
+
+			collSurplusPool.accountSurplus(_asset, _borrower, collToBorrower);
+			vestaParams.activePool().sendAsset(_asset, address(collSurplusPool), collToBorrower);
```

Comment:
After looking in the code deeper, you can simply use `singleLiquidation.collSurplus = collToLiquidate.sub(collToBorrower)`. The higher function will do the rest. It will avoid the code to do the same action twice with different value.